webui/react/src/components/kit/LineChart.tsx

```diff
@@ -162,7 +162,7 @@ export const LineChart: React.FC<LineChartProps> = ({
     return {
       axes: [
         {
-          font: '12px "Objektiv Mk3", Arial, Helvetica, sans-serif',
+          font: '12px Inter, Arial, Helvetica, sans-serif',
```

Comment:
nit: not a big deal since this is the only component where it's happening right now, but it would be nice to have font (and any other global styles) configured in one place for use in any canvas component, instead of defining it manually for each case.

Reply:
ideally I was hoping to put CSS var directly here, but I don't think it worked properly for me... probably could take another look to make sure I used the right syntax

Reply:
oh, wdy know! the css var worked great here...